### ISSUE TITLE
requires @types/command-line-usage

### ISSUE THREAD

**Issue by geofholbrook** [Roaders/ts-command-line-args#24]
I find that in order to compile a project using this package, I have to install @types/command-line-usage myself as a dev dependency. Otherwise I get this:

```
node_modules/ts-command-line-args/dist/helpers/options.helper.d.ts:1:37 - error TS7016: Could not find a declaration file for module 'command-line-usage'. '/Users/geof.holbrook/Data/repos/bridge/node_modules/command-line-usage/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/command-line-usage` if it exists or add a new declaration (.d.ts) file containing `declare module 'command-line-usage';`

1 import { Content, OptionList } from 'command-line-usage';
```

**Comment by Roaders** [comment]
Sorry I've not had a chance to look at this. I am planning on doing so. Do you have an example repo that I can have a look at? I've not had time to try and replicate it yet.

**Comment by Roaders** [comment]
should be fixed in version 2.2.1. Let me know if there are any issues.
